bitnami/airflow/README.md

```diff
@@ -256,6 +258,41 @@ airflow.clonePluginsFromGit.branch=v1.0.9-branch
 airflow.clonePluginsFromGit.path=plugins
 ```
 
+### Existing Secrets
+
+You can use an existing secret to configure your Airflow auth, external Postgres, and extern Redis passwords:
+
+```console
+postgresql.enabled=false
+externalDatabase.host=my.external.postgres.host
+externalDatabase.user=bn_airflow
+externalDatabase.database=bitnami_airflow
+externalDatabase.existingSecret=all-my-secrets
+
+redis.enabled=false
+externalRedis.host=my.external.redis.host
+externalRedis.existingSecret=all-my-secrets
+
+airflow.auth.existingSecret=all-my-secrets
+```
+
+The expected secret resource looks as follows:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: all-my-secrets
+type: Opaque
+data:
+  airflow-password: "Smo1QTJLdGxXMg=="
+  airflow-fernetKey: "YVRZeVJVWnlXbU4wY1dOalVrdE1SV3cxWWtKeFIzWkVRVTVrVjNaTFR6WT0="
+  db-password: "cG9zdGdyZXMK"
```

Comment:
The stable/postgresql chart uses `postgresql-password` for the user password.If we use the same key then user can simply specify the secret of an existing stable/postgresql chart deployment

```suggestion
  postgresql-password: "cG9zdGdyZXMK"
```

Reply:
Will do! For consistency, should we rename the `db-password` key in `secret-externaldb.yaml` to `postgresql-password` as well?